### PR TITLE
build fix and instructions

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by antoyo** [comment]
Thanks for your contribution!

I didn't know other people than me were using this web browser ;) , but I believe I don't need the log file anymore because I've fixed the issue I had (black windows issue).
So, please remove the `WriteLogger` and uncomment the `TermLogger`.
(Ideally, I should figure out why logging to syslog doesn't work from the web extension.)

Setting the environment variable is only necessary when compiling in release mode, so could be please update the instructions accordingly?
And the `//` is not needed before `== Building`: this is the start of a comment.
And to indicate a source code block in [asciidoctor](http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#source-code), please use the following syntax:
```adoc
[source,bash]
----
export ...
cd ...
----
```

By the way, what I do is setting the environment variable to `/usr/local/lib/titanium/web-extensions` and I install titanium in `/usr/local/bin`.

File: titanium-web-extension/src/lib.rs [hunk 1]
```diff
@@ -149,6 +149,7 @@ mod option_util;
 
 use std::fs::OpenOptions;
 use std::mem::forget;
+use std::path::Path;
```
Comment (antoyo) [inline titanium-web-extension/src/lib.rs]: Please remove this `use` statement as it is not required anymore.

**Comment by ghost** [comment]
I was interested in trying it out yeah :) there's an issue with the environment var still when you fresh compile without the target directories, even on debug. you may not be experiencing it because you've got it cached but try deleting the target dirs and you may run into it

**Comment by antoyo** [comment]
Oh, you're right, I'll fix that.

By the way, there's still an issue you might get when using the browser: sometimes, when you type too early in the status bar entry, the text will be reversed.
This is a known issue, and I know what's causing this issue: I just didn't have the time to fix it yet (I spent soooo much time debugging the black windows issue).
If you find any other issues (and there's many as you can see [here](https://github.com/antoyo/titanium/blob/master/src/main.rs#L23)), please open an issue.

**Pull request merged**